tuichain/api/views/loans.py

```diff
@@ -116,6 +116,58 @@ def create_loan_request(request):
     )
 
 
+@api_view(["PUT"])
```

Comment:
Is POST more adequate?

Reply:
It might be, since it's meant to be a "request" to blockchain